tests/fixtures/issues/issue-224.php

```diff
@@ -1,59 +1,6 @@
 <?php
 
 return [
-    'issue-224-A' => [
```

Comment:
This agent was giving me a bit of grief with some of the changes I made, then I looked it up in the issue tracker: https://github.com/browscap/browscap/issues/224

Turns out it was a fake agent that was later converted to a regular test when the fakes were removed in this commit: https://github.com/mimmi20/browscap/commit/d0869644bc438539e4bc14b7084b43ea1d4c50f0

The OSX platform here gets detected as "10.4" (from the string `1084`) which isn't correct (if anything it should be "10.8").  With my changes, Safari 6.0 isn't available on OSX 10.4. 

I decided to remove the test case since the agent isn't legit (I think the one below should probably removed too, but it's less of an issue).